packages/react-server/core/renderMiddleware.js

```diff
@@ -199,6 +199,9 @@ function renderPage(req, res, context, start, page) {
 
 	var timer = logger.timer("lifecycle.individual");
 
+	// e.g., http://security.stackexchange.com/a/12916
```

Comment:
> Protects some browsers (Chrome, IE) against MIME sniffing attacks.

^ That would be nice to include in the comment here.

It gives the headline so I know whether I'm interested in following the link.